webrender/src/device/gfx.rs

```diff
@@ -4252,31 +4256,33 @@ impl<B: hal::Backend> Device<B> {
         if color.is_none() && depth.is_none() {
             return;
         }
-        let mut clears = Vec::new();
-        let mut rects = Vec::new();
+        let mut color_clears = Vec::new();
+        let mut color_rects = Vec::new();
         if let Some(color) = color {
-            clears.push(hal::command::AttachmentClear::Color {
+            color_clears.push(hal::command::AttachmentClear::Color {
                 index: 0,
                 value: hal::command::ClearColor::Float(color),
             });
-            rects.push(hal::pso::ClearRect {
+            color_rects.push(hal::pso::ClearRect {
                 rect: hal::pso::Rect {
                     x: rect.origin.x as i16,
                     y: rect.origin.y as i16,
                     w: rect.size.width as i16,
                     h: rect.size.height as i16,
                 },
-                layers: 0 .. 1,
+                layers: self.bound_layer .. self.bound_layer + 1,
```

Comment:
Something isn't right here. The attachment isn't an array, the color attachment is a view into a single slice of an array. If we clear the attachment, we shouldn't specify the `bound_layer` therefore.
Perhaps, this is a bug in gfx_backend_dx12, and your code was right to begin with?